packages/extension/src/companion/CompanionMenu.tsx

```diff
@@ -17,11 +17,22 @@ import { PostBootData } from '@dailydotdev/shared/src/lib/boot';
 import CompanionContextMenu from './CompanionContextMenu';
 import '@dailydotdev/shared/src/styles/globals.css';
 import useCompanionActions from './useCompanionActions';
+import { postAnalyticsEvent } from '@dailydotdev/shared/src/lib/feed';
+import AnalyticsContext from '@dailydotdev/shared/src/contexts/AnalyticsContext';
 
 if (!isTesting) {
   Modal.setAppElement('daily-companion-app');
 }
 
+const postEventName = (
+  update: Pick<PostBootData, 'upvoted' | 'bookmarked'>,
+): string => {
+  if ('upvoted' in update) {
+    return `${!update.upvoted ? 'remove ' : ''}upvote post`;
+  }
+  return `${!update.bookmarked ? 'remove ' : ''}post bookmark`;
```

Comment:
We already have these event names somewhere. Maybe we should consider consolidating it somehow?
I'm afraid that when we change something they will not be aligned. This can be a future work as well